### PR TITLE
Added DirectVoid

### PR DESCRIPTION
**Pull request by alexw23**

### CODE REVIEW

File: src/Message/RapidDirectVoidRequest.php [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+ * eWAY Rapid Void Request
+ */
+ 
+namespace Omnipay\Eway\Message;
+
+class RapidDirectVoidRequest extends AbstractRequest
+{
+    public function getData()
+    {
+        $this->validate('amount', 'transactionReference');
```
Comment (incarnate) [inline src/Message/RapidDirectVoidRequest.php]: Only the transactionReference is required to cancel a pre-auth

**Comment by incarnate** [comment]
Perhaps the Void Message class could just be "RapidVoidRequest" instead of "RapidDirectVoidRequest" - this keeps it in line with the other generic Rapid functions (such as Capture & Refund).

Otherwise looks good!

**Comment by alexw23** [comment]
How can we go getting this merged in?

**Comment by incarnate** [comment]
@kayladnls - can this be PR reviewed for merge?

File: src/Message/RapidDirectVoidRequest.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+ 
+namespace Omnipay\Eway\Message;
+
+class RapidDirectVoidRequest extends AbstractRequest
+{
+    public function getData()
+    {
+        $this->validate('transactionReference');
+
+        $data = array();
```
Comment (kayladnls) [inline src/Message/RapidDirectVoidRequest.php]: This line isn't necessary. You could do 
```
return array('TransactionId' => $this->getTransactionReference());
```

**Comment by kayladnls** [comment]
@alexw23 I'm not seeing any tests on this? Can we get those added in, then I'd be happy to merge it in.

**Pull request closed without merging**